pkg/controllers/provisioning/v1alpha1/reallocation/terminate.go

```diff
@@ -47,3 +151,16 @@ func (t *Terminator) DeleteNodes(ctx context.Context, nodes []*v1.Node, spec *v1
 	}
 	return nil
 }
+
+
+func (t *Terminator) getNodes(ctx context.Context, provisioner *v1alpha1.Provisioner, additionalLabels map[string]string) ([]*v1.Node, error) {
```

Comment:
Can you cut an issue to dedup these two identical getNodes functions?

Reply:
Will do.